Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -484,6 +484,34 @@ class SignalProducerSpec: QuickSpec {
 				expect(error) == operationError
 			}
 		}
+		
+		describe("init(_:) base") {
```

Comment:
This should be `Result.producer`, right?

Reply:
Well technically I'm testing `SignalProducer.init(_ base:)` as well but I can call it `Result.producer`.